web/src/message_store.ts

```diff
@@ -155,7 +155,7 @@ export function clear_for_testing(): void {
     stored_messages.clear();
 }
 
-export function get(message_id: number): Message | undefined {
+export function get(message_id?: number | null): Message | undefined {
```

Comment:
This is a mistake from before, you can see in the next line that it can be undefined or null

Reply:
I can make this a prep commit

Reply:
That’s clearly an overdefensive check that should be deleted when we’re confident it’s impossible, which it ought to be: it makes no sense to want to look up a message with ID `undefined` or `null`.

However, this error does show up in Sentry exactly once, and I found the reproduction: in an empty narrow, go to user menu → **Set status**, open the emoji picker, and press Enter.

```
message_store.get got bad value 
Additional information:  
Object { message_id: null }
[blueslip.ts:34](webpack:///src/blueslip.ts)
    Logger_func blueslip.ts:34
    error blueslip.ts:100
    get message_store.ts:160
    toggle_reaction emoji_picker.js:286
    process_enter_while_filtering emoji_picker.js:318
    jQuery 8
    register_popover_events emoji_picker.js:611
    onMount emoji_picker.js:672
    invokeHook tippy.esm.js:767
    onFirstUpdate tippy.esm.js:1440
    createPopper createPopper.js:157
    (Async: promise callback)
    createPopper createPopper.js:155
    createPopperInstance tippy.esm.js:1167
    mount tippy.esm.js:1204
    show tippy.esm.js:1450
    scheduleShow tippy.esm.js:1240
    createTippy tippy.esm.js:685
    instances tippy.esm.js:1601
    tippy tippy.esm.js:1600
    toggle_popover_menu popover_menus.ts:385
    toggle_emoji_popover emoji_picker.js:691
    register_click_handlers emoji_picker.js:798
    jQuery 2
```

Reply:
Nice find. The bug there would seem to be that somehow the "Status" emoji picker is trying to do anything with accessing the message feed.

Let's do a prep commit to fix that bug.

Reply:
Should we leave the blueslip error until after the fix has been on CZO for a bit, to ensure that we've fixed the bug? Or are you confident we can fully remove the undefined/null check after that change.

Reply:
I think the evidence we’d get from leaving this on CZO for a while would be marginal, compared to the evidence we already have from there being only the one event in Sentry. I’m comfortable removing it.

Reply:
Okay! Added those changes as prep commits